test-four/main.css

```diff
@@ -7,3 +7,51 @@ body {
 }
 
 /* Place your CSS here... */
+
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  margin: 0;
+  align-content: stretch;
+}
+
+.card {
+  flex-basis: 550px;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-bottom: 20px;
+}
+
+.card:nth-of-type(3) {
```

Comment:
is this necessary? it looks like a bit of a code smell

Reply:
I wanted to get the third card element to grow larger than the others as it was presented in the brief. However looking back it is quite an ad hock approach to the problem, and does not address the underlying problem.